Phaethon/ExternalApi/Controllers/CompanyController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+using System.Web.Http.Cors;
+
+namespace ExternalApi.Controllers
+{
+    [EnableCors(origins: "http://localhost:49873", headers: "*", methods: "*")]
+    [RoutePrefix("Api/Company")]
+    public class CompanyController : ApiController
+    {
+        private readonly HttpClient _client;
+
+        public CompanyController()
+        {
+            _client = new HttpClient();
+            _client.BaseAddress = new Uri("http://localhost:64007/Company/");
+        }
+
+        [Route("GetCompanies")]
+        [HttpGet]
+        public async Task<HttpResponseMessage> GetCompanies()
+        {
+            return await _client.GetAsync("GetCompanies");
+        }
+
+        [Route("GetCompany")]
+        [HttpGet]
+        public async Task<HttpResponseMessage> GetCompany(int id)
+        {
+            var parameters = HttpUtility.ParseQueryString(string.Empty);
+            parameters["id"] = id.ToString();
```

Comment:
Just a sugestion: you could use UrlBuilder to simply switch the Port or the address or whatever you need to switch, of your Request. So you wont have to deal with reapplying parameters or any of that stuff.